### ISSUE TITLE
View service admin proposal.

### ISSUE THREAD

**Issue by davidradl** [odpi/egeria#3662]
Currently there are 2 types of view services, integration and governance services. Integration takes a list of platform orientated configuration and the governance services need a remote server omag name and platform root url , so they can communicate with an omas. I suggest:

1) We enhance the ViewServiceConfig to have ViewServiceType with enum values of Integration and governance Solution.
2) Each view service could then specify this type in ViewServiceDescription
3) This would allow us to have an admin call to start all integration view services, which would take the list config and a separate start all Governance solution call to start all governance solution calls. Each could specify mandatory parameters and have the payload as the appropriate child config. The server file could use the new type to identify which subset of view services to start.    
4) Optionally we could change the start of each service so there is a start an integration service and a start a governance solution service rather than the generic start view service call. 

What do you think?   @grahamwallis @mandy-chessell

**Comment by mandy-chessell** [comment]
I do not like the idea of combining starting a service with configuring it.  In fact it does not really make sense as actually you are starting the view server not the view service.  I think it would be better if starting the view server continues to use the standard `instance` method that all of the other OMAG Servers use rather than introducing a special approach for view servers.

In terms of configuring - I think if there are two types of service then there should be two admin services to make it clear what the request body should contain.

**Comment by grahamwallis** [comment]
I'm not sure that being able to start IntegrationViewServices separately from SolutionViewServices is necessary. As @mandy-chessell says above, we normally use the server 'instance' to start the server including all enabled services. 

I have used the terminology IntegrationViewServices and SolutionViewServices in the changes I made to the admin API configuration properties classes. I have not used the term 'Governance' there so we need to make sure we are being consistent. 

I noticed that the configuring-the-view-services.md file (checked in as part of #3629/#3630) doesn't really explain the differences between the two groups of view services or specify what configuration properties are mandatory and optional for each group. We should add more description to make this clearer I think.
